### PR TITLE
[chore] Add namedpipe receiver

### PR DESCRIPTION
**Pull request by crobert-1**
The [namedpipereceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/namedpipereceiver) is currently marked as `Alpha` stability for `logs`, and shows itself as being included in `contrib`. Adding to release for parity.

### CODE REVIEW

**Comment by crobert-1** [comment]
This needs @djaglowski's approval as code owner.

**Review by TylerHelmuth — APPROVED** [review]
Approving based on Contrib-distro guidelines of including all >= alpha components.

**Review by djaglowski — APPROVED** [review]

**Pull request merged**
